### PR TITLE
Scripts to create EML.xml and meta.xml (part of DwC-A)

### LINKED ISSUE #16: Write R pipeline for meta.xml file
The meta.xml file contains the meta data of the Darwin Core Archive files by giving the URLs of every term used as a column header. This can easily be created manually but as we want an automated and generalized way of creating this file, it should be created by a R script.

### PR DESCRIPTION
**Pull request by StefanVriend**

### CODE REVIEW

**Comment by StefanVriend** [comment]
The validation for the meta.xml works now. However, as this branch does not have the .csv versions of the DwC tables yet, the script to create the meta.xml will need to be updated when this PR merges with the main branch.

File: R/Create_EML_xml.R [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+# 3. Add attributes for specific nodes ------------------------------------
+
+# Read EML file as XML file
+EML <- xml2::read_xml(here::here("data", "EML.xml"))
+
+# Identify all taxonId nodes for which attribute shall be set
+taxonId_node <- xml2::xml_find_all(EML, xpath = "//taxonId")
+
+# Set "provider" attribute for taxonId nodes
+xml2::xml_set_attr(taxonId_node, attr = "provider", value = "https://www.gbif.org/")
```
Comment (kuytje) [inline R/Create_EML_xml.R:153]: Maybe we could be the provider here.

File: R/Create_EML_xml.R [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+# Identify title node
+title_node <- xml2::xml_find_all(EML, xpath = "//title")
+
+# Set title attribute
+xml2::xml_set_attr(title_node, attr = "xml:lang", value = "en")
+
```
Comment (kuytje) [inline R/Create_EML_xml.R:160]: If we want to be more strict to the EML schema we could add the EML namespace and location?

Example from IPT generated EML xml header

<eml:eml xmlns:eml="eml://ecoinformatics.org/eml-2.1.1"
             xmlns:dc="http://purl.org/dc/terms/"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 http://rs.gbif.org/schema/eml-gbif-profile/1.2/eml.xsd"
             packageId="http://nioo0036.nioo.int/ipt/resource?id=vogeltrekstation_historical_data_on_timing_of_ringing_of_nestling_birds/v1.4" system="http://gbif.org" scope="system"
    xml:lang="eng">

Comment (CherineJ) [reply R/Create_EML_xml.R:160]: The EML package that we use in the script to create the EML file does add that automatically to the final EML.xml file, I think. Only the "xmlIns:dc" is not included, but we also don't use any Dublin Core terms in the EML, only EML terms (if I'm correct). The package ID is a UUID that we created once and is also included already, as well as the namespace and the location. 
Should therefore be quite compliant with the EML schema already :)

File: R/Create_EML_xml.R (hunk 1 above)
Comment (StefanVriend) [reply R/Create_EML_xml.R:153]: The provider here refers to the provider of the taxon identifiers, which in this case is GBIF. We use GBIF's Taxonomic Backbone as our biological taxonomy.

**Pull request merged**